element_deeplabcut/export/nwb.py

```diff
@@ -47,7 +47,7 @@ def dlc_session_to_nwb(keys, use_element_session=True, session_kwargs=None):
         output_dir = model.PoseEstimationTask.infer_output_dir(key)
         config_file = str(output_dir / "dj_dlc_config.yaml")
         video_name = Path((model.VideoRecording.File & key).fetch1("file_path")).stem
-        h5file = str(list(output_dir.glob(f"{video_name}*h5"))[0])
+        h5file = next(output_dir.glob(f"{video_name}*h5"))
         output_path = h5file.replace(".h5", f"_{subject_id}.nwb")  # DLC2NWB convention
```

Comment:
Did you test this? `h2file` is probably of type `pathlib.Path`, which may not have a `replace` method.

Reply:
I tested the whole script, which successfully wrote out the renamed file. I copied this line from [DLC2NWB](https://github.com/DeepLabCut/DLC2NWB/blob/main/dlc2nwb/utils.py#L209)